symfony/messenger/4.3/manifest.json

```diff
@@ -3,9 +3,10 @@
         "config/": "%CONFIG_DIR%/"
     },
     "env": {
-        "#1": "To use AMQP, also run ‘composer require amqp’",
```

Comment:
I think the amqp-pack might not be necessary anymore. It installs messenger, serializer & makes sure the amqp extension is installed. In this case, we already know messenger is installed and the serializer is no longer needed. That leaves the pack as a way to give the user an error about the amqp extension if it's not installed.